workflows/lyra_sample.ll

```diff
@@ -0,0 +1,4 @@
+executable: go
+arguments:
+ - run
+ - 'github.com/thallgren/lyra-sample'
```

Comment:
It would be cool to move this to `lyraproj`.

Reply:
Yes, I agree. And perhaps give it another name too.